manifests/repos/apt.pp

```diff
@@ -10,7 +10,11 @@
   ::apt::source { $name:
     repos    => $repo,
     location => $location,
-    key      => $key,
+    key      => {
+      id     => $key,
+      source => 'http://deb.theforeman.org/foreman.asc',
+      ensure => 'refreshed',
```

Comment:
Does this mean it fetches the key on every run?

Reply:
according to https://forge.puppet.com/modules/puppetlabs/apt/reference#ensure-1:

Using 'refreshed' will make keys auto update when they have expired (assuming a new key exists on the key server).